src/plugins/plugin-knex.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+var shimmer = require('shimmer');
+var util = require('util');
+var is = require('is');
+
+var VERSIONS = '0.12.x';
```

Comment:
It looks like the current version of `knex` is 0.13.0. It would be good to extend this patching as far forward and back in version support as it can go without logic modification.